swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -969,6 +959,9 @@ module Decls {
         result.asAstNode() = ast.getPattern(j).getFullyUnresolved()
       )
       or
+      // PropertyWrapperBackingVarBinding is a special case as it shares the init
+      // with the original variable binding (see also PrintAstNode.qll)
+      ast != any(VarDecl d).getPropertyWrapperBackingVarBinding() and
```

Comment:
As we discussed in our sync. See https://codeql.github.com/docs/ql-language-reference/formulas/#equality.

```suggestion
      not ast = any(VarDecl d).getPropertyWrapperBackingVarBinding() and
```